src/main/java/javaClasses/homework_4/Lilit_Manukyan/Developer.java

```diff
@@ -0,0 +1,27 @@
+package javaClasses.homework_4.Lilit_Manukyan;
+
+public class Developer extends Staff {
+    private String techstack;
+    public Developer(String name, String lastName, String techstack) {
+        super(name, lastName);
+        this.techstack = techstack;
+    }
+
+    public String getTechstack() {
+        return techstack;
+    }
+
+    public String setTechstack(String techstack) {
+        return techstack;
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("name is: " + this.getName());
+        System.out.println("techstack is " + this.getTechstack());
+    }
+
+    public static void getCompany(){
```

Comment:
The main idea of providing line about using static method was that you should use method call from parent class, using child class (Developer.getCompany()). In this case you have two actual problems 1. Code duplication, as you have to duplicate your code from parent class in it's children classes 2. What you did here is actual attempt to override static class which is illegal